easybuild/easyconfigs/c/Cargo/Cargo-0.13.0-foss-2016b.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'Cargo'
+version = '0.13.0'
+
+homepage = 'https://crates.io/'
+description = "The Rust package manager"
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [
+    'https://github.com/rust-lang/cargo/archive/',
+    'https://github.com/rust-lang/rust-installer/archive/',
+]
+sources = [
+    '%(version)s.tar.gz',
+    '4f99485.tar.gz',  # rust-installer
+]
+
+dependencies = [
+    ('Rust', '1.12.1'),
+]
+builddependencies = [
+    ('CMake', '3.6.2'),
+    ('Python', '2.7.12'),
+    ('cURL', '7.49.1'),
+]
+
+buildopts = 'VERBOSE=1'
+# rust-installer is a git submodule, but not included in the sources pulled from GitHub
+preinstallopts = "cp -a %(builddir)s/rust-installer*/* %(builddir)s/cargo-%(version)s/src/rust-installer && "
```

Comment:
in addition, the target `rust-installer` directory already exists, I don't want to copy `rust-installer-<commit>` into `<target>/rust-installer`